doc/classes/OS.xml

```diff
@@ -196,19 +196,17 @@
 				        arguments[argument.lstrip("--")] = ""
 				[/gdscript]
 				[csharp]
-				var arguments = new Godot.Collections.Dictionary();
+				var arguments = new Dictionary&lt;string, string&gt;();
 				foreach (var argument in OS.GetCmdlineArgs())
 				{
 				    if (argument.Contains('='))
 				    {
 				        string[] keyValue = argument.Split("=");
-				        arguments[keyValue[0].LStrip("--")] = keyValue[1];
+				        arguments[keyValue[0].TrimStart('-')] = keyValue[1];
```

Comment:
Is this correct? This doesn't match the GDScript case, it expects two hyphens, I'd say the code should match

Reply:
`TrimStart` will remove any number of hyphen characters at the beginning of the string, so this should still work when there are two hyphens. However, it will now also trim any number of hyphens (one, three, ...).

If we are expecting exactly **two** hypens then it may be better to use `StringExtensions.TrimPrefix` as you suggested.

Reply:
I agree with both that TrimPrefix  / trim_prefix makes more sense here. changed it in the PR